src/core/unif.ml

```diff
@@ -33,6 +33,30 @@ let pp_problem oc p =
 let no_problems : problems =
   {to_solve  = []; unsolved = []; recompute = false}
 
+
+(** Definition of symbols used to encode unification rules. *)
+module Hint = struct
+
+  let sign =
+    let pth = [""; "unif_rule"] in
+    let s = Sign.create pth in
+    s.sign_deps := Files.PathMap.empty;
+    Sign.loaded := Files.PathMap.add pth s !(Sign.loaded);
+    s
+
+  (** Symbol representing an atomic unification problem. The term [atom t u]
+      represents [t ≡ u]. The left-hand side of a unification rule is made
+      of only one atom. *)
+  let atom : sym =
```

Comment:
I would prefer to call it, say equiv. It would be clearer as atom is very generic.